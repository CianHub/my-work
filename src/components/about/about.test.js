import React from "react";
import Aboutsection from "./about";
import renderer from "react-test-renderer";

test("Aboutsection is rendered correctly with no errors", () => {
  const component = renderer.create(<Aboutsection />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
