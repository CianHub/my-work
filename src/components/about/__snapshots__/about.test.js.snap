// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aboutsection is rendered correctly with no errors 1`] = `
<section
  id="about"
>
  <div
    className="container header"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12 text-center"
      >
        <h2
          className="section-heading"
          id="aboutText"
        >
          At Your Service
        </h2>
        <hr
          className="my-4"
          id="aboutHr"
        />
      </div>
    </div>
  </div>
  <div
    className="container about-icons"
  >
    <div
      className="row"
    >
      <div
        className="react-reveal col-lg-3 col-md-6 text-center"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="service-box mt-5 mx-auto"
        >
          <h3
            className="mb-3"
          >
            Dynamic
          </h3>
          <p
            className="text-muted mb-0"
          >
            I love making projects feel alive and vibrant to the user.
          </p>
        </div>
      </div>
      <div
        className="react-reveal col-lg-3 col-md-6 text-center"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="service-box mt-5 mx-auto"
        >
          <h3
            className="mb-3"
          >
            Responsive
          </h3>
          <p
            className="text-muted mb-0"
          >
            I like web applications to seemlessly function on any device.
          </p>
        </div>
      </div>
      <div
        className="react-reveal col-lg-3 col-md-6 text-center"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="service-box mt-5 mx-auto"
        >
          <h3
            className="mb-3"
          >
            Always Learning
          </h3>
          <p
            className="text-muted mb-0"
          >
            I enjoy learning new technologies and staying up to date with the latest tools.
          </p>
        </div>
      </div>
      <div
        className="react-reveal col-lg-3 col-md-6 text-center"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="service-box mt-5 mx-auto"
        >
          <h3
            className="mb-3"
          >
            User Friendly
          </h3>
          <p
            className="text-muted mb-0"
          >
            I aim to provide the user with best UX experience possible.
          </p>
        </div>
      </div>
    </div>
  </div>
  <br />
  <div
    className="container about-cian align-center "
  >
    <div
      className="row"
    >
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div
          className="col-lg text-center"
        >
          <img
            className="img-cian img-responsive"
            src="https://i.postimg.cc/NfRVTRBD/image.jpg"
          />
          <h3>
            Who?
          </h3>
          <hr
            className="my-4"
          />
          <p
            className="text-muted mb-0"
          >
            I'm a highly focused web developer with several years of diverse experience in a variety of digital media and design based roles. I like to create dynamic, user-friendly web applications and continue to improve my abilities.
          </p>
          <br />
          <a
            className="btn btn-info"
            href="https://drive.google.com/file/d/1XjSEkJNjIpcjNdtNM3osbvoj3HVqUCIb/view?usp=sharing"
            target="_blank"
          >
            Download My CV
          </a>
        </div>
      </div>
    </div>
  </div>
  <br />
  <div
    className="container about-progress-bars "
  >
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          HTML
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={90}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "90%",
              }
            }
          >
            <span
              className="title"
            >
              90
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          SASS
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={70}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <span
              className="title"
            >
              70
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          JavaScript
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={80}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "80%",
              }
            }
          >
            <span
              className="title"
            >
              80
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          React
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={75}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "75%",
              }
            }
          >
            <span
              className="title"
            >
              75
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          Photoshop
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={70}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <span
              className="title"
            >
              70
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          Python
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={80}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "80%",
              }
            }
          >
            <span
              className="title"
            >
              80
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          Django
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={70}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <span
              className="title"
            >
              70
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          MongoDB
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={70}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <span
              className="title"
            >
              70
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
    <div
      className="react-reveal row"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="col-lg"
      >
        <p
          className="text-muted mb-0"
        >
          PostgreSQL
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax={100}
            aria-valuemin="0"
            aria-valuenow={65}
            className="progress-bar progress-bar-animated progress-bar-striped"
            role="progressbar"
            style={
              Object {
                "width": "65%",
              }
            }
          >
            <span
              className="title"
            >
              65
              %
            </span>
          </div>
        </div>
      </div>
    </div>
    <br
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    />
  </div>
</section>
`;
